File: BLH.ApproveIQ.Portal/ClientApp/src/app/data/data/b-l-h/entities/audit-log.ts
import { Entity } from '../../../b-l-h/primitives/entity';

export class AuditLog extends Entity {
    action: string;
    entityId: string;
    entityName: string;
    oldValues: string;
    newValues: string;
    affectedColumns: string;
    createdUserId: string;
    createdOnUtc = new Date();
}

File: BLH.ApproveIQ.Domain/Entities/AuditLog.cs
using BLH.ApproveIQ.Domain.Primitives;

namespace BLH.ApproveIQ.Domain.Entities;
public class AuditLog : Entity
{
    public string? Action { get; set; }
    public Guid EntityId { get; set; }
    public string? EntityName { get; set; }
    public string? OldValues { get; set; }
    public string? NewValues { get; set; }
    public string? AffectedColumns { get; set; }
    public Guid CreatedUserId { get; set; }
    public DateTime CreatedOnUtc { get; set; }
}

